src/discordtipbot/presence_cache.cr

```diff
@@ -0,0 +1,30 @@
+# Stores all online users
+class PresenceCache
```

Comment:
I kind of think `Discord::Cache` should just be reopened for this. But meh. Either way is fine I guess. I suppose this does belong in its own store since it wraps a `Set` and doesn't actually store the discord object.